### PR TITLE
Run tests in tox environment

### PR DESCRIPTION
**Pull request by BohdanVV**
In this pull request, changes have been made to the way Python tests are run.
Previously, `coverage` was used with `xmlrunner` to run the tests.
Now, tests are being run within a `tox` environment using `pytest`.

### CODE REVIEW

**Comment by BohdanVV** [comment]
Please make sure that [this pull request in common-ci](https://github.com/51Degrees/common-ci/pull/52) is merged prior to merging the current one. @justadreamer

File: fiftyone_pipeline_core/tests/test_core.py [hunk 1]
```diff
@@ -1,49 +1,48 @@
...
-from classes.testpipeline import TestPipeline
-from classes.memorylogger import MemoryLogger
-from classes.exampleflowelement1 import ExampleFlowElement1
-from classes.exampleflowelement2 import ExampleFlowElement2
-from classes.stopflowdata import StopFlowData
-from classes.errorflowdata import ErrorFlowData
+from .classes.testpipeline import TestPipeline
+from .classes.memorylogger import MemoryLogger
+from .classes.exampleflowelement1 import ExampleFlowElement1
+# from .classes.exampleflowelement2 import ExampleFlowElement2
```
Comment (justadreamer) [inline fiftyone_pipeline_core/tests/test_core.py:30]: should we perhaps just delete these commented out imports?

File: fiftyone_pipeline_engines_fiftyone/tox.ini [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+description = run the tests with pytest
+package = wheel
+wheel_build_env = .pkg
+deps =
+    pytest>=6
+    pytest-cov
+    flask
+commands =
+    pytest --cov=. --import-mode=append {tty:--color=yes} {posargs}
+;    pytest --cov=. {tty:--color=yes} {posargs}
```
Comment (justadreamer) [inline fiftyone_pipeline_engines_fiftyone/tox.ini:15]: do we need these commented out parts, perhaps just delete them?

**Review by justadreamer — APPROVED** [review]
approving, but would be good to remove commented out lines

**Pull request merged**